src/views/AddItem.jsx

```diff
@@ -1,3 +1,29 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { useState } from 'react';
+import Form from '../components/form';
+
+export function AddItem({ addItem, listToken }) {
+	const [form, setForm] = useState({
+		itemName: '',
+		daysUntilNextPurchase: 7,
+		isSubmited: false,
+		isFailed: false,
```

Comment:
Great job on the form element! I hadn't seen this pattern before of keeping all the form's pieces in one combined state object and passing them in to a form component. I like it a lot. It's a lot cleaner than having to have change handlers for each individual input on the form. 

One question, the isFailed property in the form's state doesn't appear to be used anywhere yet. Was this put in for use in a later issue?